scopes/dependencies/pnpm/pnpm.package-manager.ts

```diff
@@ -406,6 +417,15 @@ export class PnpmPackageManager implements PackageManager {
     if (opts.componentRootDir != null) {
       filterByImporterIds.push(opts.componentRootDir as ProjectId);
     }
+    for (const importerId of filterByImporterIds) {
+      for (const depType of ['dependencies', 'devDependencies', 'optionalDependencies', 'specifiers', 'dependenciesMeta']) {
+        for (const workspacePkgName of opts.componentIdByPkgName.keys()) {
+          if (workspacePkgName !== opts.pkgName) {
+            delete lockfile.importers[importerId]?.[depType]?.[workspacePkgName]
```

Comment:
@zkochan , do you have the prettier running on commit hook? I'm wondering why it didn't add the semicolon at the end of this line

Reply:
I am not sure, isn't it configured automatically?

Reply:
it should. maybe next commit, look for something running before you get the response. if it's not working, I guess full-setup should fix it.